fastrace-jaeger/src/lib.rs

```diff
@@ -133,12 +133,12 @@ impl JaegerReporter {
 }
 
 impl Reporter for JaegerReporter {
-    fn report(&mut self, spans: &[SpanRecord]) {
+    fn report(&mut self, spans: Vec<SpanRecord>) {
         if spans.is_empty() {
             return;
         }
 
-        if let Err(err) = self.try_report(spans) {
+        if let Err(err) = self.try_report(&spans) {
```

Comment:
This seems not right

Reply:
It's intended because a span may be consumed multiple times https://github.com/fast/fastrace/blob/a6c639cbfe1ff29d0099b794a8464a486f743157/fastrace-jaeger/src/lib.rs#L116-L128